apps/shell/app.js

```diff
@@ -74,9 +72,14 @@ glob.sync(path.join((process.env.OOD_CLUSTERS || '/etc/ood/config/clusters.d'),
     if (isDefault) default_sshhost = host;
   });
 
+if (process.env.OOD_SSHHOST_ALLOWLIST) {
+  let parsed_allowlist = process.env.OOD_SSHHOST_ALLOWLIST.split(":")
+  host_allowlist = new Set(parsed_allowlist, ...host_allowlist);
```

Comment:
Used a `Spread` operator here, this allows `ENV` variables to have precedence over the allowlist parsed from `*.y*ml` but merges the configurations instead of overwriting each other.  

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Spread_syntax